### ISSUE TITLE
maya sample errors

### ISSUE THREAD

**Issue by hannesdelbeke** [hannesdelbeke/unimenu#52]
```python
import unimenu
menu = unimenu.Node(label="my menu")  # create a menu
item = unimenu.Node(label="hi", command='print("hi")')  # create a menu item
menu.items.append(item)  # add the item to the menu
menu.setup()  # setup the menu in the app, parented by default to the main menu bar
```

produces this in Maya 2022.

Errror: 'NoneType' object has no attribute 'findChild'
Traceback (most recent call last):
 File "<maya console>", line 5, in <module>
 File "c:\users\johannes\packages\unimenu\0.4.1\88a70aca30cb79a278872594adf043dc6c40af99\python\unimenu\apps\_abstract.py", line 205, in setup
parent_app_node = parent_app_node or self._default_root_parent File "c:\users\johannes\packages\unimenu\0.4.1\88a70aca30cb79a278872594adf043dc6c40af99\python\unimenu\apps\qt.py", line 48, in _default_root_parent
 menu_bar = main_window.findChild(QtWidgets.QMenuBar)
AttributeError: 'NoneType' object has no attribute 'findChild' #

_Originally posted by @nebukadhezer in https://github.com/hannesdelbeke/unimenu/issues/46#issuecomment-1629686377_

**Comment by hannesdelbeke** [comment]
similar in maya 2024
it seems the main window is `None`

new maya has main window as arnold render window, isntead of the main maya window.
the only top lvl window of type `QtWidgets.QMainWindow`

**Comment by hannesdelbeke** [comment]
when swapping to non qt, i can get sample to work by changing it to 

```
data = {"items": [{"label": "test","command": 'print("hello world")'}]}
data = {"label":"test", "items": [{"label": "test","command": 'print("hello world")'}]}
```

"label":"test",

**Comment by hannesdelbeke** [comment]
this should be resolved now, added a small hack in 0.4.4
